modules/data.land/R/match_pft.R

```diff
@@ -0,0 +1,53 @@
+##' Match model PFTs
+##' 
+##' Matches BETYdb species IDs to model-specific PFTs
+##' 
+##' @param bety_species_id  vector of BETYdb species IDs
+##' @param pfts             settings$pfts.  List of pfts with database matching based on name
+##' @param con              database connection
+##' 
+##' @return table of BETYdb PFT IDs matched to species IDs
+##' 
+match_pft <- function(bety_species_id, pfts, con){
```

Comment:
currently both ED and SIPNET `soil` PFT are associated with "unknown" as species, should we correct it on the database then? would it [a pft not being associated with any species] cause any problems elsewhere in the workflow? alternatively we can rename them as soil.ed, soil.sipnet